### PR TITLE
Clear TODOs #1

### PR DESCRIPTION
**Pull request by msaug**
Cleared some of the current TODOs in the code that could be fixed, so that they don't accumulate too much.

### CODE REVIEW

File: tests/unit_tests/protocol/libraries/logic/test_validation_logic.cairo [hunk 1]
```diff
@@ -43,30 +59,68 @@ func test_validate_withdraw_when_reserve_is_inactive{
...
+
+@external
+func test_validate_supply_when_asset_is_paused{
+    syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
+}():
+    alloc_locals
+    let (reserve) = get_test_reserve()
+    ReserveConfiguration.set_active(MOCK_A_TOKEN_1, TRUE)
+    ReserveConfiguration.set_paused(MOCK_A_TOKEN_1, TRUE)
+    %{ expect_revert(error_message="RESERVE_PAUSED") %}
```
Comment (EdgarBarrantes) [inline tests/unit_tests/protocol/libraries/logic/test_validation_logic.cairo:105]: Note: these will all fail after your other [PR](https://github.com/aave-starknet-project/aave-starknet-core/pull/108/files#diff-a63686636b2b08b81d99bbf876e4c0bc9100fa3adcd9ad66845ed3a419727741) is merged.

**Review by EdgarBarrantes — APPROVED** [review]
We need to chose which PR to merge first I guess, I'd go for the [Errors](https://github.com/aave-starknet-project/aave-starknet-core/pull/108) one. Nevertheless, see no issues here.

File: tests/test_suites/test_specs/aave_oracle_spec.cairo [hunk 2]
```diff
@@ -134,7 +134,6 @@ namespace TestAaveOracle:
     func test_non_owner_sets_ticker{
         syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
     }():
-        # TODO (depends on ACL Manager)
```
[the remark this answers is not in the document]
Comment (msaug) [reply tests/test_suites/test_specs/aave_oracle_spec.cairo:137]: Yeah the goal is just to verify if the behavior of the aave oracle contract

File: tests/unit_tests/protocol/libraries/logic/test_validation_logic.cairo [hunk 3]
```diff
@@ -76,26 +130,90 @@ func test_validate_supply_amount_null{
...
+@external
+func test_validate_drop_reserve_asset_listed_1{
+    syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr
+}():
+    alloc_locals
+    let (local reserve) = get_test_reserve()
+    # Test 1 : reserve.id != 0 => asset is listed
 
+    # update reserve.id to 1
+    let (__fp__, _) = get_fp_and_pc()
```
[the remark this answers is not in the document]
Comment (msaug) [reply tests/unit_tests/protocol/libraries/logic/test_validation_logic.cairo:152]: You need it to be able to use the `&` operator.

**Review by achab — APPROVED** [review]
Thanks a lot for cleaning all those TODOs @msaug! LGTM

**Pull request merged**